client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -0,0 +1,390 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import isEmpty from 'lodash/lang/isEmpty';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'analytics';
+import acceptDialog from 'lib/accept';
+import DisconnectJetpackDialog from 'my-sites/plugins/disconnect-jetpack/disconnect-jetpack-dialog';
+import PluginItem from 'my-sites/plugins/plugin-item/plugin-item';
+import PluginsActions from 'lib/plugins/actions';
+import PluginsListHeader from 'my-sites/plugins/plugin-list-header';
+import PluginsLog from 'lib/plugins/log-store';
+import PluginNotices from 'lib/plugins/notices';
+import SectionHeader from 'components/section-header';
+
+export default React.createClass( {
+	displayName: 'PluginsList',
+
+	mixins: [ PluginNotices ],
+
+	propTypes: {
+		plugins: PropTypes.array,
```

Comment:
> maybe just the bare mimimum props

That's what I like to see!

> What if the array is empty?

An empty array is still a valid array. This shouldn't be a problem. The problem would be sending in `null` or `42` instead of `[]`